app/models/api_token.rb

```diff
@@ -3,7 +3,7 @@ class ApiToken < ActiveRecord::Base
 
   before_create :generate_token
 
-  belongs_to :approval
+  belongs_to :approval, class_name: 'Approvals::Individual'
```

Comment:
It's needed to resolve the `has_one through:` relationship of `:user`. I bet there's a better way?